src/test/scala/pl/allegro/tech/marathon/plugin/auth/FakeHttpResponse.scala

```diff
@@ -0,0 +1,22 @@
+package tech.allegro.marathon.plugin.auth
+
+import mesosphere.marathon.plugin.http.HttpResponse
+
+class FakeHttpResponse(
```

Comment:
Good point. Right now code is very simple, but when it start growing I'll definitely take a look at some stubbing/mocking lib.